### PR TITLE
Display responsibilities for each band

### PR DESCRIPTION
**Pull request by jessicamcmillen**

### CODE REVIEW

File: ExpressAPI/db.js [hunk 1]
```diff
@@ -60,3 +60,19 @@ exports.getCompetencies = function(callback) {
         }
     )
 }
+
+exports.getResponsibilities = function(callback) {
+    db.query(
+        "select resp_band.bandID, resp_band.respID, responsibilities.respDesc FROM responsibilities INNER JOIN resp_band ON responsibilities.ID = resp_Band.respID ORDER BY bandID",
+        function (err, rows) {
+            if (err) {
+                logger.error("getResponsibilities failed with error: " + err)
```
Comment (JamesMatchett) [inline ExpressAPI/db.js]: semi colon

File: ExpressAPI/db.js [hunk 2]
```diff
@@ -60,3 +60,19 @@ exports.getCompetencies = function(callback) {
...
+                throw err;
+            }
+            logger.debug("getResponsibilities succeeded.")
```
Comment (JamesMatchett) [inline ExpressAPI/db.js]: semi colon

File: kjourney/src/models/responsibility.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export class Responsibility {
+    bandID: string
+    respID: string
+    respDesc: string
```
Comment (JamesMatchett) [inline kjourney/src/models/responsibility.ts]: 3 semi colons here please, thanks

**Review by JamesMatchett — APPROVED** [review]

File: ExpressAPI/index.js [hunk 4]
```diff
@@ -27,3 +27,9 @@ app.get('/competencies', function(req, res) {
         res.send(rows);
     })
 })
+
+app.get('/responsibilities', function(req, res) {
```
Comment (micalexis) [inline ExpressAPI/index.js]: what is the point of this route? are you not supposed to be getting responsibilities per band or role? a generic get of all responsibilities seems odd

File: kjourney/src/app/app.component.css [hunk 5]
```diff
@@ -88,7 +88,7 @@ article {
   border-radius: 15px;
 }
 
-footer {
+/* footer {
```
Comment (micalexis) [inline kjourney/src/app/app.component.css]: remove comment if not required

File: kjourney/src/app/bands/bands.component.css [hunk 6]
```diff
@@ -86,7 +86,7 @@
 	border-radius: 15px;
   }
 
-  footer {
+  /* footer {
```
Comment (micalexis) [inline kjourney/src/app/bands/bands.component.css]: same as above

File: kjourney/src/app/bands/bands.component.ts [hunk 7]
```diff
@@ -19,6 +21,10 @@ export class BandsComponent implements OnInit {
     this.data.getCompetencies().subscribe((c => {
       this.competencies = c;
     }));
+
+    this.data.getResponsibilities().subscribe((r => {
```
Comment (micalexis) [inline kjourney/src/app/bands/bands.component.ts]: 👍 but comment above should be addressed

File: kjourney/src/app/main/main.component.css [hunk 8]
```diff
@@ -86,7 +86,7 @@
 	border-radius: 15px;
   }
 
-  footer {
+  /* footer {
```
Comment (micalexis) [inline kjourney/src/app/main/main.component.css]: remove if not required

**Review by micalexis — CHANGES_REQUESTED** [review]
squash commits 
address the comments

**Review by Conormc98 — APPROVED** [review]

**Review by micalexis — APPROVED** [review]
make sure you squash the commits before merging!!!!!

File: ExpressAPI/db.js [hunk 9]
```diff
@@ -16,7 +16,7 @@ db.connect(function (err) {
         throw err;
     }
     logger.debug("Connected to MySQL.");
-})
+});
```
Comment (Conormc98) [inline ExpressAPI/db.js:19]: Big important change 😂

**Review by Conormc98 — APPROVED** [review]

**Review by e-mccoy — APPROVED** [review]

**Pull request merged**